scripts/publish-docker-images

```diff
@@ -150,6 +173,9 @@ while getopts ":d:p:s:i" opt; do
 		i)
 			IMAGE_NAME="${OPTARG}"
 			;;
+		f)
+			FILE_PATH="${OPTARG}"
```

Comment:
One of the things we kinda dropped as we moved away from the directory approach was the validation of the provided path. It's /not critical and shouldn't block this merge/, but its something that should generally be applied (to all user input!). Maybe this could be captured for a next iteration improvement?

---

Any and all checks should be performed prior to running so as to error early and not bail mid-way through the script's run (because it could interact poorly with other release processes):

* can we write to the path provided?
* can/should we continue execution if we can't write this file?
* do all the directories in the path exist? do we create them if missing?
* does this file already have content? do we overwrite it?

Again, its not critical to fix these now and I think we are circumventing the need for these checks where its used (yay `mktemp`!).

Reply:
Will create an issue to track the next iteration improvement from this comment and the comment above.